src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Alerts'
+sidebar_order: 1
+---
+
+Alerts are configured per-project and are based on the rules defined for that project. To modify the rules visit your **Project Settings** > **Alerts** > **Rules** tab. On this page you’ll see a list of all active rules and can add alert rules or modify existing rules.
+
+### Conditions
+
+Rules provide several conditions that you’re able to configure. These are fairly self explanatory and range from simple state changes (such as the defaults) to more complex filters on attributes.
```

Comment:
By contrast, "self-explanatory" does need a dash.

When we say "such as the defaults"... what do we mean? Isn't the only default alert rule "a new issue is seen"? Maybe just take out this parenthetical all together.